src/components/EventList.js

```diff
@@ -28,28 +30,28 @@ type ItemProps = { item: Event };
 const renderItem = (styles, locale, onPress, getAssetById) => ({
   item: event
 }: ItemProps) => (
-  <View style={styles.eventListItem}>
-    <TouchableOpacity
-      accessibilityTraits={["button"]}
-      accessibilityComponentType="button"
-      delayPressIn={50}
-      onPress={() => onPress(event.sys.id)}
-    >
-      <EventCard
-        name={event.fields.name[locale]}
-        locationName={event.fields.locationName[locale]}
-        eventPriceLow={event.fields.eventPriceLow[locale]}
-        eventPriceHigh={event.fields.eventPriceHigh[locale]}
-        startTime={event.fields.startTime[locale]}
-        endTime={event.fields.endTime[locale]}
-        imageUrl={`https:${
-          getAssetById(event.fields.eventsListPicture[locale].sys.id).fields
-            .file[locale].url
-        }`}
-        isFree={event.fields.isFree[locale]}
-      />
-    </TouchableOpacity>
-  </View>
+  <TouchableOpacity
```

Comment:
I've started to notice that in most native apps the touchable elements darken when you touch them rather than lighten. I chose TouchableOpacity before I noticed this, but I wonder if we should use `TouchableHighlight` instead and try and find a nice value for it to darken the card when touched.

Doesn't have to change in this PR, just a thought.

Reply:
Yeah definitely. Also I would like to use [`TouchableNativeFeedback`](https://facebook.github.io/react-native/docs/touchablenativefeedback.html) on Android.